ui/sway/include

```diff
@@ -1,6 +1,7 @@
 #!/bin/false
 export DEBS="sway i3status xfce4-terminal dmenu xwayland grim slurp brightnessctl"
 export XBPS="sway i3status xfce4-terminal dmenu xorg-server-xwayland grim slurp brightnessctl"
+export PACM="xfce4-terminal i3status sway dmenu xorg-xwayland grim slurp brightnessctl"
```

Comment:
` export PACM="sway i3status xfce4-terminal dmenu xorg-xwayland grim slurp brightnessctl"`

Reply:
That was added by @FlameKat53. I only implemented GNOME40